mixin/build.go

```diff
@@ -67,7 +67,7 @@ func buildBundles(b *builder.Builder) error {
 		template = helpers.CreateCertTemplate()
 	}
 
-	return errors.Wrap(b.BuildBundles(template, privkey, false), "Error building bundles")
+	return errors.Wrap(b.BuildBundles(template, privkey, false, false), "Error building bundles")
```

Comment:
do we want to clean here by default, since this will be hidden from the user?

Reply:
Really good question, and I think the answer is no - because iirc mixin will always increment the mix version number. If it doesn't, then this needs to be changed to true, or have mixin always bump the mixver number at the beginning to always start on a new ver.

Reply:
yep, ok.

Reply:
Ok good point, mixin only incrememnts mixver if the previous build was valid, so if it crashes it will not be valid and using bad data possibly. Changing to true.